agent/config/config.go

```diff
@@ -130,6 +130,11 @@ const (
 	//Known cached image names
 	CachedImageNamePauseContainer = "amazon/amazon-ecs-pause:0.1.0"
 	CachedImageNameAgentContainer = "amazon/amazon-ecs-agent:latest"
+
+	// DefaultCPUPeriod is set to 100 ms to set CFS period and quota for task limits
+	defaultCGroupCPUPeriod = 100 * time.Millisecond
+	maximumCGroupCPUPeriod = 100 * time.Millisecond
+	minimumCGroupCPUPeriod = 1 * time.Millisecond
```

Comment:
seems like `minimumCGroupCPUPeriod` has to be at least `8ms` to keep the the calculated task quota value within [ranges considered valid by linux](https://www.kernel.org/doc/Documentation/scheduler/sched-bwc.txt).